include/roboptim/trajectory/visualization/cubic-b-spline-matplotlib.hh

```diff
@@ -0,0 +1,134 @@
+// Copyright (C) 2015 by Félix Darricau, EPITA, AIST, CNRS.
+//
+// This file is part of the roboptim.
+//
+// roboptim is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// roboptim is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with roboptim.  If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef ROBOPTIM_TRAJECTORY_VISUALIZATION_CUBIC_B_SPLINE_HH
```

Comment:
`ROBOPTIM_TRAJECTORY_VISUALIZATION_CUBIC_B_SPLINE_MATPLOTLIB_HH` to be consistent.